dummy_oauth.sh

```diff
@@ -0,0 +1,12 @@
+echo " -------------- DUMMY OAUTH -------------- "
```

Comment:
This script seems to be a subset of `docker_e2e.sh` -- can we factor out this functionality from `docker_e2e.sh` and have `docker_e2e.sh` call this script so that we aren't repeating ourselves?